hw_device_mgr/hal/tests/base_test_class.py

```diff
@@ -42,13 +43,15 @@ def mock_halcomp(self):
 
     @pytest.fixture()
     def mock_hal(self, mocker):
-        """Fixture for mocking `hal.component`
+        """
+        Fixture for mocking `hal.component`.
 
-        This fixture patches `hal.component` and returns the mock class in
-        test function `mock_hal` argument and test object `mock_hal`
+        This fixture patches `hal.component` and returns the mock class
+        in test function `mock_hal` argument and test object `mock_hal`
         attribute.  See the `mock_halcomp` fixture for more information
         about the mock instances of this mock class.
         """
+        # This line resolves black & pep257 conflicts.  :P
```

Comment:
I have to say, I'm not at all fond of pep257 either.  It has a lot of really picky requirements about doc strings that IMO are of limited value (the above is a case in point!).

I do like `flake8`, since it helps catch real problems when test coverage is <100%, e.g. catching variables used before being set.

Anyway, this line is removed in [6f13edd](https://github.com/tormach/hw_device_mgr/pull/8/commits/6f13edd401701834257bd87815df4e16b4aae91e), and the `pep257` test is removed in [87f601b](https://github.com/tormach/hw_device_mgr/pull/8/commits/87f601b3d588f89e98792db0166d11e33e838cc8).